examples/cephfs/storageclass.yaml

```diff
@@ -20,6 +20,10 @@ parameters:
   # (optional) Ceph pool into which volume data shall be stored
   # pool: cephfs_data
 
+  # (optional) Comma seperated string of Cephfs kernel or ceph fuse
+  # mount options. Check man mount.ceph for mount options. For eg:
+  # mountOptions: readdir_max_bytes=1048576,norbytes
```

Comment:
In line 41 there is an another mountOptions parameter. What's that for?

Reply:
that will be for binding mounting staging path to target path  which will be used during node publish